### ISSUE TITLE
Error syncing changes

### ISSUE THREAD

**Issue by howardabrams** [MobileOrg/mobileorg#32]
After setting up Mobileorg to sync with Dropbox (and giving it access), clicking the Sync button on the Outlines view comes up with an error:

> **Error synching changes**  
> An error was encountered while attempting to fetch mobileorg.org from the server. The error was:  
> Unexpected error

Of course, mobileorg.org isn't responding to normal HTTP requests.

Thanks for bringing this back from the dead.

**Comment by kurtismullins** [comment]
I ran into this error as well. From reading the comments I must have missed steps in some sort of documentation. I started by letting MobileOrg access DropBox and create the folder (apps/mobileOrg).

I get the same results as the screenshot above. Perhaps a better error message or better documentation in the app, itself, would be useful? (I haven't fixed the issue locally yet so I don't know that the comments are 100% accurate)
